packages/hardhat/contracts/YourContract.sol

```diff
@@ -62,26 +62,26 @@ contract YourContract is Ownable {
         return (builderStream.cap * (block.timestamp - builderStream.last)) / FREQUENCY;
     }
 
-    function addBuilderStream(address payable _builder, uint256 _cap, address _optionalTokenAddress) public onlyOwner {
-        streamedBuilders[_builder] = BuilderStreamInfo(_cap, block.timestamp - FREQUENCY, _optionalTokenAddress);
-        emit AddBuilder(_builder, _cap);
+    function addBuilderStream(address payable _builder, uint128 _cap, address _optionalTokenAddress) public onlyOwner {
+        streamedBuilders[_builder] = BuilderStreamInfo(_cap, uint128(block.timestamp - FREQUENCY), _optionalTokenAddress);
```

Comment:
If I call this function directly, no event is fired. 

Maybe if you don't want this function to be called from outside, the **public** should be removed.